Tests/pageObject/RadioNavigationalWarningsAdminListEndUser.page.ts

```diff
@@ -40,7 +40,7 @@ export default class RadioNavigationalWarningsListEndUser
     public async verifyTableDateColumnData()
     {
       const resultYear= await this.page.$$eval('[id^="DateTimeGroupRnwFormat"]' , (matches: any[]) => { return matches.map(option => option.textContent.trim().slice(-2)) });
```

Comment:
Is there another way to do this other than $$eval?  As per https://playwright.dev/docs/api/class-page#page-eval-on-selector-all this is not the recommended way.

Reply:
will resolved in other branch not relevant for the this branch

Reply:
@nevillejrbrown if it's okay I'll merge this as is and perhaps you could raise this a a general point with Prasad & Jeegar separately.